client/src/app/components/how-it-works/how-it-works.component.html

```diff
@@ -0,0 +1,44 @@
+<div style="direction: rtl;">
```

Comment:
Remove the `direction`, there should be a direction for the whole website (you can see it [here](https://github.com/hasadna/open_pension/blob/master/client/src/styles.scss#L6))